services/shhext/service.go

```diff
@@ -92,6 +105,16 @@ func (s *Service) Stop() error {
 	return nil
 }
 
+// apiByNamespace retrieve an assigned api by namespace or returns nil.
+func (s *Service) apiByNamespace(namespace string) interface{} {
```

Comment:
It can be a function with the following signature: `func apiByNamespace(apis []rpc.API, namespace string) rpc.API`. It's pretty general and could be put in `services/` directory.